README.md

```diff
@@ -0,0 +1,21 @@
+# It's ALIVE!
+It's ALIVE! is a single-player VR experience in which you play the Doctor's Assistant. Your goal: to bring the doctor's creation to life!
+
+## About
+
+What does this do? You can go more here. Any inspiration? Info dump :D
+
+## Getting Started
+Hey! I want to run this on my PC, how do I do it? 
+
+In order to develop this game, you will need:
+
+Unity 2019.4 (.13 or higher, but check later)
+Universal Render Pipeline
+Shader Graph
+
+(i recommend doing one OS first, then if you want compatibility go back later later)
+### Pre-requisites (
+- do I need anything? Any one-click downloads ie unity? Any specific ver?
+### Instructions
+steps
```

Comment:
If it's helpful, you can add a TODO section. This let's folks know what you're planning to work on. It doesn't have to be everything or every little detail, but it can include like `Write out detailed setup instructions` or `Check version compatibility`